src/30_Advanced/all-videos-supported-by-amp.html

```diff
@@ -0,0 +1,115 @@
+<!--
+  #### Introduction
+
+  A showcase off all the video types supported by AMP
```

Comment:
Suggestion: "There many different ways to integrate videos in AMP HTML files. Here is a list of all currently supported video formats and platforms."